### PR TITLE
Translate zh/engines/table-engines/special/distributed: sync translate

### PR DESCRIPTION
**Pull request by ghost**
Changelog category (leave one):

- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by cnmade** [comment]
@evillique can this pull request be done?

File: docs/zh/engines/table-engines/special/distributed.md [hunk 1]
```diff
@@ -1,31 +1,127 @@
...
+SETTINGS
+    fsync_after_insert=0,
+    fsync_directories=0;
+```
+
+Data will be read from all servers in the `logs` cluster, from the `default.hits` table located on every server in the cluster.
+Data is not only read but is partially processed on the remote servers (to the extent that this is possible).
+For example, for a query with `GROUP BY`, data will be aggregated on remote servers, and the intermediate states of aggregate functions will be sent to the requestor server. Then data will be further aggregated.
+
+Instead of the database name, you can use a constant expression that returns a string. For example: `currentDatabase()`.
```
Comment (evillique) [inline docs/zh/engines/table-engines/special/distributed.md:96]: This segment seems to be mostly in English, is it supposed to be this way?

Comment (cnmade) [reply docs/zh/engines/table-engines/special/distributed.md:96]: I will update is soon

**Comment by cnmade** [comment]
translate has been update

**Review by evillique — APPROVED** [review]

**Pull request merged**